disco/extensions/upgrade_simulation/upgrades/thermal_upgrade_functions.py

```diff
@@ -50,8 +52,8 @@ def correct_line_violations(
                 options = pd.DataFrame([options])  # convert to required DataFrame format
                 options = options.rename_axis(deciding_property_list)  # assign names to the index
             options = options.reset_index().sort_values("normamps")
-            if row["max_per_unit_loading"] > 3:  # i.e. equipment is very overloaded, then oversize more to avoid multiple upgrade iterations
-                row["required_design_amp"] = row["required_design_amp"] * 1.25
+            if row["max_per_unit_loading"] > 2.25:  # i.e. equipment is very overloaded, then oversize more to avoid multiple upgrade iterations

```

Comment:
Where does 2.25 come from? Should this be a constant? Should it be configurable?

Reply:
Doesn't have to be user-configurable, cause we arrived at this factor based on some observations.

But have now changed it to be defined as a variable in the function itself, with a comment on what that parameter means, and how we arrived at it. 

 So could be redefined (if needed) later.